### DISCUSSION TITLE
Girslite only seems to transmit the intro

### DISCUSSION THREAD

**Discussion by luppahannu** [bengtmartensson/AGirs#62] [General]
Hey!

I've been using irScrutinizer with Girslite flashed to an Arduino Nano, with instructions from harctoolbox.org. The capturing of the signals seems to work like a charm, but I ran into problems transmitting some signals.

Some code types have an intro sequence and then the repeat sequence, and when trying to send these the Nano only seems to output the intro, and then stops and send back "OK". I've checked the pin output with a scope. I tested out connecting to the Nano with a terminal program (using Realterm) and sending it "send" commands. If I give the original "send" command (send 1 38000 2 34 0 <durations>)  the output is the same as with trying to transmit fron irScrutinizer, only the intro gets sent (so it basically just goes "high" once). But if make the intro sequence 0 length and add 2 to the repeat length (send 1 38000 0 36 0 <durations>) the whole signal gets sent (and the device I'm trying to control gets the command). Same thing if I make the intro length 36.

First I tried this with the girlsLite 1.0.2 hex that is linked in the harctoolbox.org, and after painstakingly getting the 1.0.3 compiled and flashed to the Nano, also with that one.

I'm not sure if this is a bug or a user error. I haven't used github that often so I'm unsure if this is the correct place to post this.

**Comment by bengtmartensson** [comment]
This is by design. If sending a signal _n_ times, and the intro is non-empty, first the intro is sent, and then _n-1_ copies of the repeat. (If the intro is empty, instead _n_ copies of the repeat are sent.)

In IrScrutinizer, just raise the "Count" on the "Sending hw" pane.

If you think this needs to be better documented, I am open to you suggestion.

> I haven't used github that often so I'm unsure if this is the correct place to post this.

It is.

**Reply by luppahannu** [reply]
Ah, I feel a bit silly now, I must have skimped over that part of the instructions numerous times.

I don't think there's anything to fix in the AGirs instructions, but maybe in irScrutinizer it would be beneficial for the user to know this relationship between the "count" setting and the sending of the intro and repeat sequences.

**Reply by bengtmartensson** [reply]
You are right. I will add it to the IrScrutinizer manual to the next release. Thank you for bringing this to my attention.

**Comment by bengtmartensson** [comment]
Please open a new issue, since this is unrelated to the original issue. Then enable the "verbose" option (GUI: Options -> Verbose), perform the experiment, and post the content of the console.
